### PR TITLE
[HackaDoc] Narayans changes

### PR DESCRIPTION
**Pull request by NarayanAnnamalai**
[Hackadoc] Adding article for Vnet Scenario, setting up a DMZ in Azure

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NarayanAnnamalai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: articles/virtual-network-scenario-set-up-DMZ.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
+<properties 
+   pageTitle="virtual-networks-scenario-set-up-DMZ" 
+   description="Demonstrate how to setup a DMZ application in Azure using Virtual Network and Network Security Group"
```
Comment (mollybostic) [inline articles/virtual-network-scenario-set-up-DMZ.md]: Make this a complete sentence (using sentence casing) and, if space allows, spell out DMZ--that will help with SEO.

File: articles/virtual-network-scenario-set-up-DMZ.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
+<properties 
+   pageTitle="virtual-networks-scenario-set-up-DMZ"
```
Comment (mollybostic) [inline articles/virtual-network-scenario-set-up-DMZ.md]: This needs to be a friendly title, as it's the link text that shows up in search results

File: articles/virtual-network-scenario-set-up-DMZ.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+<tags
+   ms.service="virtual-network"
+   ms.devlang=""
+   ms.topic="article"
+   ms.tgt_pltfrm="na"
+   ms.workload="infrastructure-services" 
+   ms.date="02/20/2015"
+   ms.author="narayan"/>
+
+# Setting up DMZ in Azure
```
Comment (mollybostic) [inline articles/virtual-network-scenario-set-up-DMZ.md]: Spell out what DMZ stands for--that will help with SEO

**Comment by azurecla** [comment]
__@NarayanAnnamalai__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: articles/virtual-network-scenario-set-up-DMZ.md (hunk 2 above)
Comment (NarayanAnnamalai) [inline articles/virtual-network-scenario-set-up-DMZ.md]: Done

File: articles/virtual-network-scenario-set-up-DMZ.md (hunk 3 above)
Comment (NarayanAnnamalai) [inline articles/virtual-network-scenario-set-up-DMZ.md]: DMZ is an industry term and spelling it out will not convey the intended meaning

**Comment by tysonn** [comment]
Narayan - the file name needs to be all lowercase, and the paths to the images need to match. The publishing tool won't publish articles that contain capital letters.

**Comment by tysonn** [comment]
Closing per conversation with Cheryl.

**Pull request closed without merging**